config/autoload.php

```diff
@@ -24,6 +24,8 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+define('_PS_VERSION_', '1.7.0.0');
```

Comment:
Are you sure this should be hardcoded here? This _define_ is generated on installation based one this one:
https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/install_version.php